app/views/admin/workload-points/workload-points.html

```diff
@@ -0,0 +1,307 @@
+            <table>
+              <tbody>
+                <tr>
+                  <th scope="row">SDR</th>
+                  <td class="numeric">132</td>
+                </tr>
+                <tr>
+                  <th scope="row">FDR (SDR Conversion)</th>
+                  <td class="numeric">41</td>
+                </tr>
+                <tr>
+                  <th scope="row">PAROM</th>
+                  <td class="numeric">123</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+        
+        <br>
+        <p><span class="bold">Weighting</span></p>
+        <div class="grid-row">
+          <div class="column-two-thirds">
+            <table>
+              <tbody>
+                <tr>
+                  <th scope="row">Overdue Terminations</th>
+                  <td class="numeric">0%</td>
+                </tr>
+                <tr>
+                  <th scope="row">Active Warrants</th>
+                  <td class="numeric">0%</td>
+                </tr>
+                <tr>
+                  <th scope="row">Only UPW Outstanding</th>
+                  <td class="numeric">100%</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+        
+        <br>
+        <p><span class="bold">Nominal Target (Points)</span></p>
+        <div class="grid-row">
+          <div class="column-two-thirds">
+            <table>
+              <tbody>
+                <tr>
+                  <th scope="row">PO</th>
+                  <td class="numeric">2176</td>
+                </tr>
+                <tr>
+                  <th scope="row">PSO</th>
+                  <td class="numeric">2176</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+       
+        <br>
+        <p><span class="bold">Default Contracted Hours</span></p>
+        <div class="grid-row">
+          <div class="column-two-thirds">
+            <table>
+              <tbody>
+                <tr>
+                  <th scope="row">PO</th>
+                  <td class="numeric">37</td>
+                </tr>
+                <tr>
+                  <th scope="row">PSO</th>
+                  <td class="numeric">37</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+      </div>      
+
+    </div>
+  </div>
+
+  {% if edit %}
+    <div class="notice">
+      <i class="icon icon-important"></i>
+      <strong class="bold-small">
+        Saving changes made here will cause reports to be updated. The report update process can take 30 minutes to complete, in which time no further changes can be saved.
+      </strong>
+    </div>
+    <br>  
+    <div class="form-group">
+      <input class="button" type="submit" value="Save">
+    </div>
+  {% else %}
+    <a href="/admin/workload-points/edit" class="button">Edit</a>
```

Comment:
This edit button appears at the bottom of every table.  It brings back to the custody tab